src/App.tsx

```diff
@@ -109,7 +109,7 @@ const Download = ({ app }: { app: PIXI.Application }) => {
         ${!app ? "visibility: hidden" : null};
       `}
       title="download your dingo"
-      href=""
+      href="#fixme"
```

Comment:
Builds fail because of a lint error here. I gave up fixing it

Reply:
😭